jekyll/_cci2/api-developers-guide.md

```diff
@@ -0,0 +1,562 @@
+
+    | Parameter    | Description                                                             |
+    | ------------ | ----------------------------------------------------------------------- |
+    | `VCS`        | Version Control System - either `github`/`gh` or `bitbucket`/`bb`       |
+    | `org-name`   | Organization name, or your personal username to your VCS                |
+    | `repo-name`  | Name of your project repo                                               |
+    | `job-number` | The number for the job you want to download artifacts from - see step 2 |
+    {: class="table table-striped"}
+    
+    You should get a list of artifacts back - if the job you selected has artifacts associated with it. Here's an extract from the output when requesting artifacts for a job that builds these docs:
+
+    ```
+    {
+      "path": "circleci-docs/assets/img/docs/walkthrough6.png",
+      "node_index": 0,
+      "url": "https://53936-48750547-gh.circle-artifacts.com/0/circleci-docs/assets/img/docs/walkthrough6.png"
+    },
+    {
+      "path": "circleci-docs/assets/img/docs/walkthrough7.png",
+      "node_index": 0,
+      "url": "https://53936-48750547-gh.circle-artifacts.com/0/circleci-docs/assets/img/docs/walkthrough7.png"
+    },
+    {
+      "path": "circleci-docs/assets/img/docs/walkthrough8.png",
+      "node_index": 0,
+      "url": "https://53936-48750547-gh.circle-artifacts.com/0/circleci-docs/assets/img/docs/walkthrough8.png"
+    },
+    ```
+
+4. Next, you may extend this API call to download the artifacts. Navigate to the location you would like to download the artifacts to, and run the following command, remembering to substitute your own values in the request:
+
+     ```sh
+    curl -X GET https://circleci.com/api/v2/project/{VCS}/{org-name}/{repo-name}/{job-number}/artifacts \
+    -H 'Content-Type: application/json' \
+    -H 'Accept: application/json' \
+    -H "Circle-Token: $CIRCLECI_TOKEN" \
+    | grep -o 'https://[^"]*' \
+    | wget -v -i -
+    ```
+
+    `Note: `grep` is used to locate all the URLs for downloading the job artifacts, while `wget` is used to perform the download.
+
+
+# Reference
+
+## API Endpoints
+
+The CircleCI v2 API, and its associated endpoints allow you to make HTTP calls to designated endpoints developed in the underlying CircleCI API architecture. These endpoints provide programmatic access to CircleCI services, including pipelines, workflows, and jobs. 
+
+Before working with the API and making calls, you should first have an understanding of the various API requests you can make to API v2 endpoints. The sections below describe these API requests.
+
+### User Endpoints
+
+| Endpoint               | Description                       
+|------------------------|-------------------------------------------------------
+| GET /user/{id}         | This endpoint enables you to return information about a specific user by making a GET request and passing the `id` parameter in the request |
```

Comment:
@thomasr8 I stuck `GET` in for both these user endpoints as it wasn't there before and seemed inconsistent with the other tables. Just wanted to check with you if it's actually correct :-)

Reply:
@rosieyohannan Yes, that is correct! Good catch 👍